platformio.ini

```diff
@@ -1241,6 +1241,29 @@ build_flags       = ${common_stm32.build_flags}
 extra_scripts     = ${common.extra_scripts}
   pre:buildroot/share/PlatformIO/scripts/generic_create_variant.py
 
+#
+# Anet ET4-MB_V1.x/ET4P-MB_V1.x (STM32F407VGT6 ARM Cortex-M4)
+# For use with with davidtgbe's OpenBLT bootloader https://github.com/davidtgbe/openblt/releases
+# Comment out board_build.offset = 0x10000 if you don't plan to use OpenBLT/flashing directly to 0x08000000.
+#
+[env:Anet_ET4_OpenBLT]
+platform             = ${common_stm32.platform}
+extends              = common_stm32
+build_flags          = ${common_stm32.build_flags} -DHAL_SD_MODULE_ENABLED -DHAL_SRAM_MODULE_ENABLED
+board                = genericSTM32F407VGT6
+board_build.core     = stm32
+board_build.variant  = MARLIN_F4x7Vx
+board_build.ldscript = ldscript.ld
+board_build.firmware = firmware.srec
```

Comment:
If your end goal is SD based flashing, this will be named "ET4.BIN"

Reply:
No idea if it's ET5.bin on the ET5 machines, but if so, you can take an approach similar to ledge where all of this is an an ANET_ETX_BASE and the different environments override the firmware name.

Reply:
> If your end goal is SD based flashing, this will be named "ET4.BIN"

It will be, but this will use OpenBLT for now since I knew using the stock bootloader wouldn't be possible, or at least not easily.

Reply:
> No idea if it's ET5.bin on the ET5 machines

It depends on the printer. Example: `et4.bin`, `et4p.bin`, `et4x.bin`, `et5.bin`, `et5p.bin`, `et5x.bin`.